src/sentry/rules/processing/delayed_processing.py

```diff
@@ -39,13 +40,20 @@
 EVENT_LIMIT = 100
 
 
-class UniqueCondition(NamedTuple):
+class UniqueConditionQuery(NamedTuple):
+    """
+    Represents all the data that uniquely identifies a condition class and its
+    single respective Snuba query that must be made. Multiple instances of the
+    same condition class can share the single query.
+    """
+
     cls_id: str
     interval: str
     environment_id: int
+    comparisonInterval: str | None = None
 
-    def __repr__(self):
-        return f"id: {self.cls_id},\ninterval: {self.interval},\nenv id: {self.environment_id}"
+    # def __repr__(self):
```

Comment:
Is this supposed to be in here?

Reply:
oh whoops I removed that to make debugging easier, will change it back